spacy_llm/tasks/lemma/task.py

```diff
@@ -0,0 +1,90 @@
+from typing import Any, Callable, Dict, Generic, Iterable, List, Optional, Type
+
+import jinja2
+from spacy.language import Language
+from spacy.scorer import Scorer
+from spacy.tokens import Doc
+from spacy.training import Example
+
+from ...ty import FewshotExample, TaskResponseParser
+from ..util import ExampleType, SerializableTask
+from .util import DEFAULT_LEMMA_TEMPLATE_V1
+
+
+class LemmaTask(SerializableTask[ExampleType], Generic[ExampleType]):
+    def __init__(
+        self,
+        template: str = DEFAULT_LEMMA_TEMPLATE_V1,
+        parse_responses: Optional[TaskResponseParser] = None,
+        fewshot_example_type: Optional[Type[FewshotExample]] = None,
+        examples: Optional[List[ExampleType]] = None,
+    ):
+        """Default lemmatization task.
+
+        template (str): Prompt template passed to the model.
+        parse_responses (Optional[TaskResponseParser]): Callable for parsing LLM responses for this task.
+        examples (Optional[Callable[[], Iterable[Any]]]): Optional callable that
+            reads a file containing task examples for few-shot learning. If None is
+            passed, then zero-shot learning will be used.
+        """
```

Comment:
`fewshot_example_type` isn't documented in the docstring

Reply:
Fixed in https://github.com/explosion/spacy-llm/pull/242/commits/047a496ca863c972e2a35600b6492960cac81706.